PGM/src/main/java/tc/oc/pgm/menu/gui/SettingsTypeInterface.java

```diff
@@ -0,0 +1,94 @@
+package tc.oc.pgm.menu.gui;
+
+import com.google.api.client.util.Lists;
+import me.anxuiz.settings.Setting;
+import me.anxuiz.settings.SettingManager;
+import me.anxuiz.settings.Toggleable;
+import me.anxuiz.settings.bukkit.PlayerSettings;
+import me.anxuiz.settings.bukkit.plugin.Permissions;
+import me.anxuiz.settings.types.BooleanType;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.entity.Player;
+import tc.oc.commons.bukkit.gui.buttons.Button;
+import tc.oc.commons.bukkit.gui.buttons.empty.EmptyButton;
+import tc.oc.commons.bukkit.gui.interfaces.SinglePageInterface;
+import tc.oc.commons.bukkit.tokens.TokenUtil;
+import tc.oc.commons.bukkit.util.Constants;
+import tc.oc.commons.bukkit.util.ItemCreator;
+import tc.oc.pgm.PGM;
+import tc.oc.pgm.PGMTranslations;
+import tc.oc.pgm.mutation.Mutation;
+import tc.oc.pgm.mutation.MutationMatchModule;
+import tc.oc.pgm.mutation.command.MutationCommands;
+import tc.oc.pgm.picker.PickerSettings;
+import tc.oc.pgm.tokens.gui.MainTokenMenu;
+import tc.oc.pgm.tokens.gui.MutationConfirmInterface;
+
+import java.util.*;
+
+public class SettingsTypeInterface extends SinglePageInterface {
+
+    public SettingsTypeInterface(Player player) {
+        super(player, new ArrayList<>(), 27, "Setting Types");
+        update();
+    }
+
+    @Override
+    public void setButtons() {
+        List<Button> buttons = new ArrayList<>();
+        buttons.add(new Button(
+                new ItemCreator(Material.BOOK_AND_QUILL)
```

Comment:
GUI object `Button` only takes an `ItemCreator` object as "icon" parameter I think? Out of curiosity, what are the advantages of using one or the other?